### PR TITLE
MQ135 Gas sensor added

### LINKED ISSUE #1912: Add support for MQ 135 gas sensor
**Actual Behaviour**
Currently no feature to get output of MQ 135 sensor

**Expected Behaviour**
Add support for MQ 135

**Steps to reproduce it**
N/A

**LogCat for the issue**
N/A

**Screenshots of the issue**
N/A

**Would you like to work on the issue?**
Yes

@CloudyPadmal since this is not a I2C sensor, I was thinking of adding it as a separate instrument like barometer and luxmeter. Does that sound okay?

### PR DESCRIPTION
**Pull request by neel1998**
Fixes #1912 
**Changes**:Support for MQ135 added

Steps : 
1. Connect V of MQ135 to VDD 
2. Connect GND of MQ135 to GND
3. Connect AO of MQ135 to CH1

**Screenshot/s for the changes**:
https://drive.google.com/file/d/1-KxOaqE_Y5EYquMkebYpBOEc0d7GAdLS/view?usp=sharing

**Checklist**: [Please tick following check boxes with `[x]` if the respective task is completed]
- [x] I have used resources from `strings.xml`, `dimens.xml` and `colors.xml` without hard-coding them
- [x] No modifications done at the end of resource files `strings.xml`, `dimens.xml` or `colors.xml`
- [x] I have reformatted code in every file included in this PR [<kbd>CTRL</kbd>+<kbd>ALT</kbd>+<kbd>L</kbd>]
- [x] My code does not contain any extra lines or extra spaces
- [x] I have requested reviews from other members

**APK for testing**: 
[mq135.zip](https://github.com/fossasia/pslab-android/files/3473731/mq135.zip)

@CloudyPadmal  @mariobehling @cweitat have a look. For now i haven't added the logging, because that would have made this PR too huge. I have also asked @ho-dor if he can help me with svg logo for the gas sensor. The basic functionality is working fine as can be seen in the video

### CODE REVIEW

File: app/src/main/java/io/pslab/fragment/InstrumentsFragment.java [hunk 1]
```diff
@@ -161,7 +166,8 @@ protected Void doInBackground(Void... params) {
                     R.string.compass_description,
                     R.string.gyroscope_description,
                     R.string.thermometer_desc,
-                    R.string.robotic_arm_descriptoin
```
Comment (CloudyPadmal) [inline app/src/main/java/io/pslab/fragment/InstrumentsFragment.java]: Correct this one's spelling along with this

File: app/src/main/res/values/strings.xml [hunk 2]
```diff
@@ -849,6 +849,11 @@
 
     <string name="no_baro_sensor">Device does not have a barometer</string>
 
+    <string name="gas_sensor">Gas Sensor</string>
+    <string name="gas_sensor_description">Detects amounts of gases in air</string>
```
[the remark this answers is not in the document]
Comment (CloudyPadmal) [reply app/src/main/res/values/strings.xml]: >Air Quality Sensor (MQ135) Description: Air quality sensor for detecting a wide range of gases, including NH3, NOx, alcohol, benzene, smoke and CO2

File: app/src/main/java/io/pslab/fragment/InstrumentsFragment.java (hunk 1 above)
Comment (neel1998) [reply app/src/main/java/io/pslab/fragment/InstrumentsFragment.java]: Sorry for such blunders!. I type fast and then such mistakes get out of my attention. :joy:

**Comment by neel1998** [comment]
@CloudyPadmal  yes I am doing it... making fixes is a while

**Comment by neel1998** [comment]
@CloudyPadmal  done. We can ignore Codacy for now, since most of it will be solved once logging is added to gas sensor

**Pull request merged**